d2l-dom-focus.js

```diff
@@ -1,275 +1,20 @@
-import './d2l-dom.js';
-import './d2l-dom-visibility.js';
+import * as core from '@brightspace-ui/core/helpers/focus.js';
```

Comment:
I don't have an amazing argument for this, but this feels a little like `SELECT *` in SQL. If stuff gets added later, you'll get it imported... which is probably not what you want here? I'm assuming this repo will likely get frozen/archived and no new stuff will get added to it, so we don't want it to pick up extra stuff.

Reply:
Yeah you're right.  There is some risk there.  I'll switch it to be explicit.